cocoa/events_darwin.go

```diff
@@ -46,6 +47,10 @@ func (w *Window) EventChan() (events <-chan interface{}) {
 	downKeys := make(map[string]bool)
 	ec := make(chan interface{})
 	go func(ec chan<- interface{}) {
+
+		noX := -1
+		lastXY := image.Point{-1, -1}
```

Comment:
hmmm... this technically works as is but is crap.  I was in a rush to fix it and should have paid better attention.  Fix coming soon.